tests/run_tests.sh

```diff
@@ -2,11 +2,15 @@
 
 set -eo pipefail
 
-export NOMAD_VERSION=0.11.2
+export NOMAD_VERSION=1.0.2
 export CONTAINERD_VERSION=1.3.4
 export PATH=$PATH:/usr/local/go/bin
 export PATH=$PATH:/usr/local/bin
-export GOPATH=/home/circleci/go
+if [ -e /home/circleci ]; then
```

Comment:
Is there a case where `/home/circleci` does not exist and we have to explicitly set `GOPATH=$HOME/go`?

Reply:
same as above

Reply:
This is okay. We can leave it as part of this PR.